components/select/index.ts

```diff
@@ -1,6 +1,8 @@
 import Select from './select'
 import SelectOption from './select-option'
+import useSelectHandle from './use-imperactive-select'
```

Comment:
the file name should be `use-select-handle`

Reply:
done